### PR TITLE
Feat : Add alerts

### PR DESCRIPTION
**Pull request by OmKakatkar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,44 +15,9 @@ <h1 class="text-center">Components</h1>
...
-                <h3>Filled</h3>
-                <section class="container flex-container">
-                    <button class="btn bg-blue">Sharp Edge</button>
-                    <button class="btn bg-green">Sharp Edge</button>
-                    <button class="btn bg-red">Sharp Edge</button>
-                    <button class="btn bg-purple">Sharp Edge</button>
-                </section>
-                <h3>Border Only</h3>
-                <section class="container flex-container">
-					<button class="btn bd-blue">Sharp Edge</button>
```
Comment (surajmore98) [inline index.html:39]: issue with code format

File: index.html [hunk 2]
```diff
@@ -15,44 +15,9 @@ <h1 class="text-center">Components</h1>
...
-					<button class="btn bd-green">Sharp Edge</button>
-					<button class="btn bd-red">Sharp Edge</button>
-					<button class="btn bd-purple">Sharp Edge</button>
-				</section>
-                <h3>Disabled</h3>
-                <section class="container flex-container">
-                    <button class="btn" disabled>Sharp Edge</button>
-                </section>
-				</section>
```
Comment (surajmore98) [inline index.html:48]: extra section tag.

Comment (OmKakatkar) [reply index.html:48]: This code has been removed from this file. See the `-` sign on the right as well as the red color

File: index.html (hunk 1 above)
Comment (OmKakatkar) [reply index.html:39]: These html files are going to be deleted eventually and replaced with a documentation site. So, not an issue right now. Thanks for catching the error though

Comment (surajmore98) [reply index.html:39]: no issue then

File: index.html (hunk 2 above)
Comment (surajmore98) [reply index.html:48]: i completly missed that.

**Review by surajmore98 — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (OmKakatkar) [reply index.html:48]: Thanks for catching this error again

**Comment by Hnh013** [comment]
hi Om , i'll suggest use alert-success instead of alert success as success could be a generic term for styling for your future components

**Review by Hnh013 — APPROVED** [review]
nothing critical or breaking to observe , hence approving, good to merge !

**Pull request merged**